TF_Service_dotNet/TouchFree/ServiceTypes.cs

```diff
@@ -116,91 +116,6 @@ public WebsocketInputAction(InputAction _data)
         }
     }
 
-    public static class Utilities
```

Comment:
Did you mean to just remove these, I thought you said they were moving to a new file.